cylc/flow/scripts/set.py

```diff
@@ -177,7 +177,7 @@ def get_option_parser() -> COP:
     return parser
 
 
-def validate_tokens(tokens_list: Tuple['Tokens']) -> None:
+def validate_tokens(tokens_list: Tuple['Tokens', ...]) -> None:
```

Comment:
Had to look up the meaning of `...` in this context, curious.

This function doesn't actually *require* a tuple, it just needs something it can iterate over, so this might make more sense:

```suggestion
def validate_tokens(tokens_list: Iterable['Tokens']) -> None:
```

It's generally a good idea to use the loosest description possible in method signatures to give yourself more wiggle room later, e.g. with `Iterable` you could provide a generator instead of a tuple for efficiency reasons.

Note, requires import change on line 93.